mmv1/templates/terraform/examples/network_services_mesh_basic.tf.erb

```diff
@@ -0,0 +1,9 @@
+resource "google_network_services_mesh" "<%= ctx[:primary_resource_id] %>" {
+  provider    = google-beta
+  name        = "<%= ctx[:vars]['resource_name'] %>"
+  labels      = {
+    foo = "bar"
+  }
+  description = "my description"
+  interception_port = 443
```

Comment:
Given `interception_port` is optional could you also provide an example without it?

Reply:
Added!